resources/asciidoctor/lib/elastic_compat_preprocessor/extension.rb

```diff
@@ -150,8 +150,9 @@ def reader.process_line(line)
         line = super
         return nil if line.nil?
 
-        if SOURCE_WITH_SUBS_RX =~ line
-          line.sub! "subs=\"#{$1}\"", "subs=\"#{$1},callouts\"" unless $1.include? 'callouts'
+        SOURCE_WITH_SUBS_RX.match(line) do |m|
+          old_subs = m[1]
+          line.sub! "subs=\"#{old_subs}\"", "subs=\"#{old_subs},callouts\"" unless old_subs.include? 'callouts'
```

Comment:
Some people may disagree but I would prefer `()` here for readability. 

```ruby
line.sub!("subs=\"#{old_subs}\"", "subs=\"#{old_subs},callouts\"") unless old_subs.include?('callouts')
```

(For what it's worth, I use `()` everywhere but that's a much larger debate)

Also, what's the reason you're checking whether `old_subs` contains the substring `callouts`?

Reply:
> Also, what's the reason you're checking whether `old_subs` contains the substring `callouts`?

I'll leave a comment. This is a difference between Asciidoctor and AsciiDoc. AsciiDoc will add `callouts` to the list automatically. Asciidoctor doesn't. But we have zillions of places where we assume that it *is* added automatically.